Units/parser-c.r/bug1458930.c.d/expected.tags

```diff
@@ -1,2 +1,2 @@
-x	input.c	/^char x();$/;"	p	file:
-y	input.c	/^wchar_t y();$/;"	p	file:
+x	input.c	/^char x();$/;"	p
```

Comment:
Yep.. maybe it's better to try to follow the old rules, so we remain compatible with whatever tool is using that feature.

So all prototypes go in file scope? Or only the ones declared in *.c files?